src/constants.ts

```diff
@@ -1,2 +1,7 @@
-export const MOCK_MODULES_BLACKLIST = ['react', 'prop-types', 'styled-components'];
-export const IMPORT_MODULES_BLACKLIST = ['prop-types', 'styled-components'];
\ No newline at end of file
+export const FILE_SUFFIX = process.env.JEST_TEST_GEN_FILESUFFIX;
```

Comment:
this is not a constant, it is a configuration, no need to read the config inside this file, it can be done in main.ts